imitation/scripts/config/train.py

```diff
@@ -89,9 +89,18 @@ def swimmer():
 
 
 @train_ex.named_config
-def debug():
+def fast():
+    """Minimize the amount of computation. Useful for test cases."""
     n_epochs = 1
     interactive = False
     n_disc_steps_per_epoch = 1
     n_gen_steps_per_epoch = 1
     n_episodes_per_reward_data = 1
+
+
+@train_ex.named_config
```

Comment:
Does this really need to be a named config (only sets a single parameter, just used in tests)? Consider using `config_updates` keyword argument in `run` instead.

Reply:
good point